docs/userguide/Clients.md

```diff
@@ -116,6 +116,26 @@ To set default HTTP headers:
 $client->setDefaultOption('headers/X-Custom-Header', 'FooBar');
 ```
 
+## User agents
+
+php-opencloud will send a default `User-Agent` header for every HTTP request, unless a custom value is provided by the end-user. The default header will be in this format:
+
+> OpenCloud/xxx cURL/yyy PHP/zzz
+
+where `xxx` is the current version of the SDK, `yyy` is the current version of cURL, and `zzz` is the current PHP version. To override this default, you must run:
+
+```php
+$client->setUserAgent('MyCustomUserAgent');
```

Comment:
I think it would be helpful to show the user agent that will be sent over the wire in this case and the next (prefix) one.